src/main/java/Duke.java

```diff
@@ -1,10 +1,218 @@
+            Task task = new Deadline(taskName,date);
+            tasks.add(task);
+            System.out.println(ADDING_TASK);
+            System.out.println("[D][] " + taskName + " (by: " + date + ")");
+            System.out.println("Now you have " + tasks.size() + " tasks in the list.");
+        } else {
+            System.out.println(INVALID_INPUT);
+        }
+    }
+
+    private static void addEventTask(String content, ArrayList<Task> tasks) {
+        String[] seperatedWordsInContent = content.split(" /");
+        if (handleException(seperatedWordsInContent,2)) {
+            return;
+        }
+        String beginDate = seperatedWordsInContent[1].split(" ", 2)[1];
+        if (handleException(seperatedWordsInContent[2].split(" ", 2),1)) {
+            return;
+        }
+        String endDate = seperatedWordsInContent[2].split(" ", 2)[1];
+
+        if (seperatedWordsInContent[1].startsWith("from ") && seperatedWordsInContent[2].startsWith("to ")) {
+            String taskName = seperatedWordsInContent[0];
+            Task task = new Event(taskName,beginDate,endDate);
+            tasks.add(task);
+            System.out.println(ADDING_TASK);
+            System.out.println("[E][] " + taskName + " (from: " + beginDate + " to: " + endDate + ")");
+            System.out.println("Now you have " + tasks.size() + " tasks in the list.");
+        } else {
+            System.out.println(INVALID_INPUT);
+        }
+    }
+
+    private static void changeTaskStatus(ArrayList<Task> tasks, String[] seperatedWords) {
+        try {
+            int lastWordInInteger = Integer.parseInt(seperatedWords[1]);
+            if (lastWordInInteger > tasks.size()) {
+                System.out.println(BIG_MARK_NUMBER);
+            } else {
+                boolean doesCommandContainsUnmark = seperatedWords[0].equals("unmark");
+                if (doesCommandContainsUnmark) {
+                    tasks.get(lastWordInInteger - 1).unMark();
+                    System.out.println(FINISH_UNMARKING_MESSAGE);
+                } else {
+                    tasks.get(lastWordInInteger - 1).mark();
+                    System.out.println(FINISH_MARKING_MESSAGE);
+                }
+                System.out.println(lastWordInInteger + ". " + "[" + tasks.get(lastWordInInteger - 1).getMarkingStatus()
+                        + "]" + tasks.get(lastWordInInteger - 1).getContent());
+            }
+        } catch (Exception error) {
+            System.out.println(MARK_INSTRUCTION + "\n" + seperatedWords[0] + ": Number");
+        }
+    }
+
+    private static void printItems(ArrayList<Task> container) {
+        for (int i = 0; i < container.size(); ++i) {
+            Task item = container.get(i);
+            if (item == null) {
+                break;
+            } else if (item.getClassSymbol().equals("T")) {
+                System.out.println((i + 1) + ". " + "[" + item.getClassSymbol() + "]" + "["
```

Comment:
Perhaps it could be more readable if "[" and "]" is returned alongside item.getClassSymbol()? such that for Task.java the method could be as such

```
public String getMarkingStatus() {
        return isMarked ? "[X]" : "[ ]";
    }
```